portfolio.css

```diff
@@ -0,0 +1,137 @@
+body {
```

Comment:
[ OPTIONAL ] The page has a horizontal scroll. It means that some content is overflowing out of its container. Please fix the scroll as it is not considered good for the design.
![scroll](https://user-images.githubusercontent.com/35504149/133620134-39596ec6-e2d0-44b3-bd04-b87c62e2b028.png)